assets/queries/cloudFormation/ecs_task_defenition_healthcheck_missing/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ECS_task_defenition_healthcheck_missing",
+  "queryName": "ECS Task Defenition HealthCheck Missing",
+  "severity": "LOW",
+  "category": "Operational",
+  "descriptionText": "The HealthCheck property specifies an object representing a container health check",
```

Comment:
The description should explain more about the vulnerability and not the property itself. Try something like "Amazon ECS must have the HealthCheck property defined to give more control over monitoring the health of tasks."